commands/slash/resume.js

```diff
@@ -25,7 +25,7 @@ const command = new SlashCommand()
         embeds: [
           new MessageEmbed()
             .setColor("RED")
-            .setDescription("There is no song playing right now."),
+            .setDescription("Please join a voice channel and run this command again"),
```

Comment:
```suggestion
            .setDescription("There is no song playing right now."),
```
This is still incorrect, revert or fix please.
https://github.com/SudhanPlayz/Discord-MusicBot/pull/918#discussion_r901953425